Emby.Dlna/PlayTo/DeviceInfo.cs

```diff
@@ -1,64 +0,0 @@
-#pragma warning disable CS1591
```

Comment:
Renamed DeviceInfo.cs file with ToDeviceIndentification() removed as it's no longer required.